### DISCUSSION TITLE
Diff between smugmug stats and file downloaded

### DISCUSSION THREAD

**Discussion by patrikrask** [tommyblue/smugmug-backup#48] [Q&A]
Hi
Whne I visit my smugmug stats page it states that I have ~100 000 uploads but when I count the files after doing a smugmug-backup I see only ~9500 files that has been downloaded. Do anyone knows where the diff comes from?

**Comment by tommyblue** [comment]
@patrikrask do you mean 100k (100_000) and 9.5k (9_500)? so you're missing 90500 files? it seems a bit too much. Didn't the log show anything?
Did you also check the metadata file?

**Comment by tommyblue** [comment]
I didn't know about the stats page 😄 
For me, it shows 62153 files and the metadata file has 62157 rows, while I have 66780 files on disk. Those are little differences caused by files deleted in smugmug but not locally, I'm pretty sure everything is fine. Can you make those checks yourself? 🙏🏼

**Comment by patrikrask** [comment]
Hi
I meant 95 000 vs 100 000

tis 23 jan. 2024 kl. 18:52 skrev Tommaso Visconti ***@***.***
